package-lock.json

```diff
@@ -1,14 +1,35 @@
 {
   "name": "mocha-circleci-reporter",
-  "version": "0.0.2",
+  "version": "0.0.3",
```

Comment:
it's usually a mistake to update the version in a pr

Reply:
`npm` did that. Do you want me to undo it?